cargo_all

```diff
@@ -2,8 +2,8 @@
 
 (cd fastpbkdf2 && cargo +nightly $*)
 (cd octavo && cargo +nightly $*)
-(cd openssl && cargo +nightly $*)
 (cd ring && cargo +nightly $*)
 #un-maintained(cd dagenix && cargo +nightly $*)
 (cd RustCrypto && cargo +nightly $*)
 (cd sodiumoxide && cargo +nightly $*)
+(cd openssl && cargo +nightly $*)
```

Comment:
Why reorder the tests? Please add a commit in this file about why the order is significant, if it is.